data/young2020epidemiologic/young2020epidemiologic-extraction.md

```diff
@@ -0,0 +1,67 @@
+# Extraction of Young et al. (2020) using the data published by Challenger et al. (2022) and Goyal et al. (2020)
+
+[Young et al. 2020]((https://jamanetwork.com/journals/jama/fullarticle/2762688)) collect clinical, laboratory, and radiologic data, including PCR cycle threshold values from nasopharyngeal swabs and viral shedding in blood, urine, and stool. The clinical course was summarized, including the requirement for supplemental oxygen, intensive care, and the use of empirical treatment with lopinavir-ritonavir. The numbers of positive stool, blood, and urine samples were small. Data for the nasopharyngeal swab results were obtained from the combined dataset in the supplementary materials of Challenger et al. BMC Medicine (2022) 20:25 (https://doi.org/10.1186/s12916-021-02220-0). The standard curve was calculated based on the concentration provided by Goyal et al. (2020)(https://www.science.org/doi/10.1126/sciadv.abc7112). 
+
+First, we `import` python modules needed:
+
+```python
+#import modules;
+import yaml
+import pandas as pd
+import numpy as np
+from shedding_hub import folded_str, literal_str
+```
+
+Raw data ([CombinedDataset](https://github.com/shedding-hub/shedding-hub/blob/main/data/young2020epidemiologic/CombinedDataset.xlsx)), which is stored on [Shedding Hub](https://github.com/shedding-hub), will be loaded and cleaned to match the most updated [schema](https://github.com/shedding-hub/shedding-hub/blob/main/data/.schema.yaml).
+
+```python
+df = pd.read_excel(os.path.join(current_directory, 'Challenger2022/CombinedDataset.xlsx'), sheet_name='Viral_Load')
+young2020 = df[df['StudyNum'] == 6].copy()
+columns_to_drop = ['Estimated', 'SevMax', 'Sev1st', 'Died', 'SevMax3']
+young2020 = young2020.drop(columns=columns_to_drop)
+# Transform Ct value to gc/swab using obtained standard curve
```

Comment:
Consider adding some descriptions of how the standard curve was back-calculated and maybe adding a visualization (points with the standard curve from Goyal data, Viral_Loads.csv) here in the Markdown file. Write down the equation of the standard curve in a Markdown block and explain this is obtained from Goyal et al. (2020).

Reply:
Just to confirm, if I want to add an image to a .md file, I can embed an image link using a format similar to how we include links for datasets. I can link to an image uploaded to the 'young2020epidemiologic' folder. Or is there a specific place where you would like me to store the image or a preferred naming convention for the image file, or is there a better way to include images? Please let me know 😃.

Reply:
Yes, I think you need to include the figure in the  'young2020epidemiologic' folder and insert
`![plot](plot.png)`
in your md code.